p2p/types/network.go

```diff
@@ -1,16 +1,15 @@
 package types
 
-import (
-	"fmt"
-	"sort"
+//go:generate mockgen -source=$GOFILE -destination=./mocks/network_mock.go github.com/pokt-network/pocket/p2p/types Network
```

Comment:
How did you discover this? Didn't cross my mind to search for something along these lines.

Reply:
🙂 When I saw that moving files broke tests because `mockgen` didn't generate the expected mock anymore I thought about another project I contributed to where `go generate` is used in all sorts of ways and I just adapted the same concept here.